src/pages/dataset-registration-page/form-informationmodel/external-information-models/external-information-models.component.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import InputField from '../../../../components/fields/field-input/field-input.component';
+import MultilingualField from '../../../../components/multilingual-field/multilingual-field.component';
+import LinkReadonlyField from '../../../../components/fields/field-link-readonly/field-link-readonly.component';
+import InputFieldReadonly from '../../../../components/fields/field-input-readonly/field-input-readonly.component';
+import { datasetFormPatchThunk } from '../../formsLib/asyncValidateDatasetInvokePatch';
+import { getConfig } from '../../../../config';
+
+import '../form-informationmodel.component.scss';
+
+const ExternalInformationModels = ({
+  fields,
+  titleLabel,
+  linkLabel,
+  addLabel,
+  languages,
+  isReadOnly,
+  dispatch,
+  catalogId,
+  datasetId
+}) => {
+  const onDeleteFieldAtIndex = (fields, index) => {
+    const values = fields.getAll();
+    // use splice instead of skip, for changing the bound value
+    values.splice(index, 1);
+    const patch = { [fields.name]: values };
+    const thunk = datasetFormPatchThunk({ catalogId, datasetId, patch });
+    dispatch(thunk);
+  };
+
+  const isFdkURI = (uri) => uri && uri.includes(`${getConfig().searchHost}/informationmodels/`);
```

Comment:
Hva er hensikten med denne?

Reply:
Er et hack for å skille de to måtene vi legger til modellene.

De som legges til vha søk skal ligge som pills i den øvre delen og de som legges til vha lenke skal vises i den nedre listen. Men det er kun regisreringsløsningen som bryr seg om at de registreres på to måter. De lagres i samme array mot backend